test/dlc_tests/conftest.py

```diff
@@ -1029,6 +1031,45 @@ def _validate_pytorch_framework_version(request, image_uri, test_name, skip_dict
     return False
 
 
+@pytest.fixture(scope="session")
+def telemetry():
+    telemetry_test = os.getenv("TEST_TYPE", "UNDEFINED")
```

Comment:
nit: no need to put string default with bool(True) value, you should just be able to leave as `test_pytorch_training_job_type_env_var`

Can also refactor to
```
is_telemetry_test = os.getenv("TEST_TYPE") == "telemetry"
if is_mainline_context() and is_telemetry_test:
   ...
```

Reply:
sorry I'm a bit confused by what you mean by 

> nit: no need to put string default with bool(True) value, you should just be able to leave as test_pytorch_training_job_type_env_var